compilers/concrete-compiler/compiler/Makefile

```diff
@@ -85,6 +87,15 @@ else
 		CXX_COMPILER_OPTION=
 endif
 
+# If the build type is Debug, and the lld linked is installed
```

Comment:
It would be helpful to mention in the README that the build environment can use `lld` if it's installed. Otherwise, it might be difficult to discover for non-Zama people.

Reply:
ok, I added a paragraph